pkg/cmd/util/clientcmd/factory.go

```diff
@@ -205,7 +205,7 @@ func NewFactory(clientConfig kclientcmd.ClientConfig) *Factory {
 		if err != nil {
 			return defaultMapper, defaultTyper
 		}
-
+		////--
```

Comment:
Ah forgot to take this out, it was a quick way of marking where I left off in that file.